app.js

```diff
@@ -0,0 +1,118 @@
+/** @format */
+
+'use strict';
+{
+  // 要素を取得
+  const btnsDiv = document.getElementById('buttons');
+  const btnStart = document.querySelector('.btnStart');
+  const head = document.querySelector('.header');
+  const question = document.querySelector('.question');
+  const genre = document.querySelector('.genre');
+  const difficult = document.querySelector('.difficult');
+
+  // 変数を定義
+  let val = 0;
+  let correctVal = 0;
+
+  // 開始ボタンのイベント
+  btnStart.addEventListener('click', () => {
+    getQuiz(val);
+  });
+
+  // 非同期処理
+  function getQuiz(val) {
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(val);
+      }, 1000);
+
+      head.textContent = '取得中';
+      question.textContent = '少々お待ちください。';
+      btnsDiv.removeChild(btnStart);
+    }).then(() => {
+      callAPI(val);
+    });
+  }
+
+  // API呼び出し
+  async function callAPI(val) {
+    const response = await fetch(
+      'https://opentdb.com/api.php?amount=10&type=multiple'
+    );
```

Comment:
API の呼び出しはネットワークエラーなど予期しない例外が起こることが想定されるので
基本的に `try/catch` をしておきましょう。

```suggestion
    let response;
    try {
      response = await fetch('https://opentdb.com/api.php?amount=10&type=multiple');
    } catch (e) {
      console.error(e);
    }
```